CONTRIBUTING.md

```diff
@@ -52,6 +52,35 @@ We welcome your contributions. There are many ways you can help us. This is few
 
 Before you submit a new PR, make you to run `npm test`. Do not submit a PR if tests are failing. If you need any help, create an issue and ask.
 
+## Issue Triage
+
+If you are looking for a way to help the project, triaging issues is a great place to start. Here's how you can help:
+
+### Responding to issues
+
+Issues that are tagged `question` or `requires-reproduction` are great places to help. If you can answer a question, it will help the asker as well as anyone searching. If an issue needs reproduction, you may be able to guide the reporter toward one, or even reproduce it yourself using the [technique above](https://github.com/storybooks/react-storybook/blob/master/CONTRIBUTING.md#reproductions).
+
+### Triaging issues
+
+Once you've helped out on a few issues, if you'd like triage access, you can help label issues, and respond to reporters. New issues should be labelled as one of `bug`, `enhancement`, `question` or `discussion`.
+
+If an issue is a `bug`, if it doesn't have a clear reproduction that you have personally confirmed, mark it `requires-reproduction` and ask the author to try and create a reproduction, or have a go yourself, if you can.
```

Comment:
> that has been confirmed

I think we either need to use a `reproduced` label (that you add when you personally confirm something) or the person that marks something as a bug should label it `requires-reproduction` if they haven't confirmed it themselves. It's easy for things to slip through the cracks and not get confirmed otherwise.

I like the `requires-reproduction` label because helping people reproduce things is a good beginner task.